llvm/include/llvm/ProfileData/SampleProf.h

```diff
@@ -618,7 +623,7 @@ class SampleContext {
   void clearState(ContextStateMask S) { State &= (uint32_t)~S; }
   bool hasContext() const { return State != UnknownContext; }
   bool isBaseContext() const { return FullContext.size() == 1; }
-  StringRef getName() const { return Name; }
+  ProfileFuncRef getName() const { return Name; }
```

Comment:
I guess this should be `getId()` now (or however we end up naming the `ProfileFuncRef` now).